components/utility.css

```diff
@@ -232,6 +233,8 @@
 .d_inline {
   display: inline;
 }
+
 .d_inlineblock {
   display: inline-block;
 }
+
```

Comment:
I think There should be better naming like d_inline_block so that its more readable. Rest all good.